framework/test/unit/modules/token/constants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export const AMOUNT = BigInt('100000000');
+export const AVAILABLE_BALANCE = BigInt('10000000000');
+export const ESCROW_AMOUNT = BigInt('100000000000');
+export const INVALID_ADDRESS = '1234';
+export const INVALID_TOKEN_ID = '00';
+export const INVALID_CHAIN_ID = '00';
+export const NON_SUPPORTED_TOKEN_ID = '8888888888888888';
+export const TOTAL_SUPPLY = BigInt('100000000000000');
```

Comment:
i think these doesn't make much sense outside of the particular context

Reply:
I think having these "magic" numbers outside the tests (in a constants file or otherwise) enables them to be re-used and updated without having to update the test itself.

Reply:
some of the constants might make sense (for example, invalid tokenID or chainID), but available balance, amount etc depends on the context / what they are testing, so i think it would be hard to be reused